src/partition/repart.c

```diff
@@ -1634,8 +1649,14 @@ static int context_dump_partitions(Context *context, const char *node) {
                 char uuid_buffer[ID128_UUID_STRING_MAX];
                 const char *label;
 
+                const char *activity = NULL;;
+
                 if (p->dropped)
                         continue;
+                else if (p->current_size == UINT64_MAX)
```

Comment:
`else if` → `if`. The if checks are entirely unrelated after all, and given the `continue` the first branch is final in this block anyway...